alestojanovic/scavenger-hunt-v1
# feat: added 404 page, refactored styling

### What did I do?
Added 404 page, refactored submit button, and few general styling changes.

### How to test?
Either run locally or check how it looks on:
https://alestojanovic.github.io/scavenger-hunt-main/
Add random url parameter to land on 404 page.
